docs/visual-testing/workflows/review.md

```diff
@@ -99,14 +99,14 @@ The list of snapshots can be grouped and filtered by using the "Group by and "Fi
-| ------------------- | ---------------------------------------------------------------------------------------------------- | --------------------------------------------------------- |
-| Suite Name          | The Name of the Suite/Class                                                                          | This will automatically be determined by the used binding |
-| Test Name           | The Name of the Test that holds the visual checks                                                    | This will automatically be determined by the used binding |
-| Device              | The Device that has been used to take the snapshot. This can be a Desktop browser or a Mobile Device | This will automatically be determined by the used binding |
-| OS                  | The OS that has been used to take the snapshot.                                                      | This will automatically be determined by the used binding |
-| Storybook (Depth 1) | The name of the Storybook library you used.                                                          | This will automatically be determined by the used binding |
-| Storybook (Depth 2) | The name of the Storybook component that was used to create the snapshot.                            | This will automatically be determined by the used binding |
+| Group by            | Description                                                                                          | Determined by                                              |
+|:--------------------|:-----------------------------------------------------------------------------------------------------|:-----------------------------------------------------------|
+| Suite Name          | The Name of the Suite/Class                                                                          | This will automatically be determined by the used binding  |
+| Test Name           | The Name of the Test that holds the visual checks                                                    | This will automatically be determined by the used binding  |
+| Device              | The Device that has been used to take the snapshot. This can be a Desktop browser or a Mobile Device | This will automatically be determined by the used binding  |
+| OS                  | The OS that has been used to take the snapshot.                                                      | This will automatically be determined by the used binding  |
+| Storybook (Depth 1) | The name of the Storybook library you used.                                                          | This will automatically be determined by the used binding  |
+| Storybook (Depth 2) | The name of the Storybook component that was used to create the snapshot.                            | This will automatically be determined by the used binding  |
 
 #### Diff Review-overlay
```

Comment:
Can we change this to "Diff Review Overlay", pls?
I was removing all those dashes before but have missed this one, it seems.